accepted/00000-http-api.md

```diff
@@ -0,0 +1,214 @@
+- Feature Name: HTTP API
+- Start Date: 2022-02-08
+
+# Summary
+
+Provide an HTTP API exposing the existing methods via plain JSON, in addition to the XMLRPC protocol.
+
+
+# Motivation
+
+Reports suggest that SUSE Manager is seeing more use in automated scenarios, where it is a part of a bigger system and driven via its APIs - possibly but not necessarily a CI system.
+
+The XMLRPC protocol has a very low barrier of entry for some use cases, notably Python programs, but as of recent years the industry has favored REST and, in general, HTTP APIs in terms of popularity and tooling support.
+
+
+# Detailed Design
+
+The design aims to provide a plain HTTP endpoint to the existing methods by reusing the available business logic with minimum redundancy. The bulk of the effort is to decouple the existing layers from protocol-dependent parts as much as possible, so that the most of the workflow can be utilized for both interfaces in an agnostic manner.
+
+![Current workflow](images/http-api-before.png)
+*Current workflow*
+
+The components marked with red are tightly coupled to the XMLRPC-specific functionality, and cannot be reused with HTTP as-is. In contrast, the green parts are agnostic, and can be used for both interfaces.
```

Comment:
Nitpick

On my screen i do not see anything in "green". It looks just black

Reply:
Fixed :+1: